backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5637900_sys_gh11302_process_remove_old_hu.sql

```diff
@@ -0,0 +1,131 @@
+INSERT INTO m_hu (ad_client_id, ad_org_id, created, createdby, isactive, m_hu_id,m_hu_pi_version_id, value, updated, updatedby, hustatus)
+VALUES (1000000,1000000,date('2022-05-05'),100,'Y',100,101,100.,date('2022-05-05'),100,'A');
+
+CREATE OR REPLACE FUNCTION remove_hu_entries_between_dates(
```

Comment:
the function name is misleading. please pick a clearer one.
also, please add a comment that explains with a lot of exclamation marks what this function does